mod-kb-ebsco/mod-kb-ebsco-java.postman_collection.json

```diff
@@ -16746,6 +16758,11 @@
 													"    pm.expect(tv4.missing.length).to.equal(0, \"Missing schemas: \" + JSON.stringify(tv4.missing));",
 													"});",
 													"",
+													"//Check that added tag exists",
```

Comment:
seems this is a duplication block of what is written on the line below - https://github.com/folio-org/folio-api-tests/pull/90/files#diff-287b4edf4a93d1a7b77ffca48ce81928R16835

Reply:
Thanks @natali-zaitseva 
Fixed.